src/form/Form.jsx

```diff
@@ -676,9 +676,14 @@ function Form({
 
       // Add a repeated row if the value went from unset to set
       // And this is the last field in a set of repeated fields
+      const isPreviousValueDefaultArray =
```

Comment:
were we just not handling multi file fields properly before?

Reply:
basically any array based field

Reply:
also maybe checking is a random value is an empty array should be a helper function in utils